--- ekiga-3.2.7/lib/engine/components/libnotify/libnotify-main.cpp~	2010-05-30 23:58:01.000000000 +0200
+++ ekiga-3.2.7/lib/engine/components/libnotify/libnotify-main.cpp	2011-04-07 13:22:56.466190747 +0200
@@ -158,7 +158,11 @@
 
   notif = notify_notification_new (notification->get_title ().c_str (),
 				   notification->get_body ().c_str (),
-				   urgency, NULL);
+				   urgency
+#if !defined(NOTIFY_VERSION_MINOR) || (NOTIFY_VERSION_MAJOR == 0 && NOTIFY_VERSION_MINOR < 7)
+				   , NULL
+#endif
+				   );
 
   g_signal_connect (notif, "closed",
 		    G_CALLBACK (on_notif_closed), notification.get ());
--- ekiga-3.2.7/src/gui/main.cpp~	2010-05-30 23:58:01.000000000 +0200
+++ ekiga-3.2.7/src/gui/main.cpp	2011-04-07 13:23:45.542858431 +0200
@@ -2820,7 +2820,11 @@
 
   body = g_strdup_printf ("%s\n%s\n%s", uri, app, account);
   
-  notify = notify_notification_new (title, body, GM_ICON_LOGO, NULL);
+  notify = notify_notification_new (title, body, GM_ICON_LOGO
+#if !defined(NOTIFY_VERSION_MINOR) || (NOTIFY_VERSION_MAJOR == 0 && NOTIFY_VERSION_MINOR < 7)
+		  , NULL
+#endif
+		  );
   notify_notification_add_action (notify, "accept", _("Accept"), notify_action_cb, mw, NULL);
   notify_notification_add_action (notify, "reject", _("Reject"), notify_action_cb, mw, NULL);
   notify_notification_set_timeout (notify, NOTIFY_EXPIRES_NEVER);
