diff -ur ekiga-2.0.2-o/ekiga.desktop.in.in ekiga-2.0.2/ekiga.desktop.in.in
--- ekiga-2.0.2-o/ekiga.desktop.in.in	2006-03-22 05:59:58.000000000 -0700
+++ ekiga-2.0.2/ekiga.desktop.in.in	2006-06-06 12:35:50.000000000 -0600
@@ -9,4 +9,4 @@
 Icon=@PACKAGE_NAME@
 Terminal=false
 Type=Application
-Categories=GNOME;GTK;Network;Telephony;
+Categories=GTK;GNOME;Network;InstantMessaging;
Only in ekiga-2.0.2: ekiga.desktop.in.in~
